catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/CswSourceConfiguration.java

```diff
@@ -92,7 +103,11 @@ public String getPassword() {
     }
 
     public void setPassword(String password) {
```

Comment:
Better yet, I'll update the EncryptionService to return an empty string if an empty string is passed in the decrypt method.